sql/moz-fx-data-shared-prod/fenix_derived/google_ads_campaign_cost_breakdowns_v1/query.sql

```diff
@@ -0,0 +1,76 @@
+WITH campaigns AS (
+  SELECT DISTINCT
+    id,
+    name,
+    FORMAT("%s (%s)", name, CAST(id AS string)) AS fenix_compatible_campaign_name
+  FROM
+    `moz-fx-data-bq-fivetran`.google_ads.campaign_history
+),
+install_dou_metrics AS (
+  SELECT
+    fenix_marketing_metrics.adjust_campaign AS fenix_marketing_metrics_adjust_campaign,
+    fenix_marketing_metrics.submission_date AS date,
+    COALESCE(SUM(fenix_marketing_metrics.new_profiles), 0) AS new_profiles_sum,
+    COALESCE(SUM(fenix_marketing_metrics.dau), 0) AS dau_sum
+  FROM
+    `moz-fx-data-shared-prod.fenix.marketing_attributable_metrics` AS fenix_marketing_metrics
+  GROUP BY
+    fenix_marketing_metrics_adjust_campaign,
+    date
+)
+SELECT
+  campaigns.name AS campaign_name,
+  stats.date AS date,
+  -- Total spend per-campaign
+  stats.cost_micros AS campaign_spend_in_microcents,
+  -- Impressions and clicks over time for each campaign (clicks of our ads)
+  impressions AS ad_impressions,
+  install_dou_metrics.new_profiles_sum AS installs,
+  install_dou_metrics.dau_sum AS dous,
+  clicks AS ad_clicks,
```

Comment:
let's call this `revenue_generating_ad_clicks` to differentiate from ads we run and their clicks